File: Quest/Assets/Src/Cards/TournamentCard.cs
using UnityEngine;
using System.Collections;


public class TournamentCard : StoryCard
{
    public int reward;

    public TournamentCard(string name, int reward) : base(name, CardTypes.Tournament)
    {
        this.reward = reward;
    }

    public int getReward(int participants) { return reward + participants; }

    public override void doEffect(GameState gState)
    {
        gState.startTournament(this);
    }
}

File: eclipse/Quest/src/model/cards/TournamentCard.java
package model.cards;

import model.game.GameState;

public class TournamentCard extends StoryCard {

	public int reward;

    public TournamentCard(String name, int reward)
    {
    	super(name, CardTypes.Tournament);
        this.reward = reward;
    }

    public int getReward(int participants) { return reward + participants; }

    public void doEffect(GameState gState)
    {
        gState.startTournament(this);
    }
}
